documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -437,6 +437,21 @@ method. Specifically, these annotations will be found if they are directly prese
 indirectly present, or meta-present on a given element.
 ====
 
+[[writing-tests-conditional-execution-custom]]
+==== Custom Conditions
+
+A container or test may be enabled or disabled based on the boolean return of a static
```

Comment:
We should describe that the method does not have to be static in case `@TestInstance(PER_CLASS)` is used and add tests for this case.

Reply:
Should I add a code sample to illustrate this and/or the usage of an external method?